### ISSUE TITLE
Pediatric investigational plan number needs to be addressed. 

### ISSUE THREAD

**Issue by BSnoeijerCD** [cdisc-org/DDF-RA#420]
Pediatric investigational plan number needs to be addressed. It is more a reference than an identifier. 
See EMA template https://www.ema.europa.eu/en/documents/template-form/paediatric-investigation-plan-pip-key-elements-guidance_en.pdf

Child tickets:

| Item  | Ticket |
| ------------- | ------------- |
| Model |  #435 |
| CT |  #434 |
| API | #436 |
| Rules | #438 |
| IG | #437 |
| Test Data | #439  |

**Comment by BSnoeijerCD** [comment]
@dih-cdisc @EMuhlbradt @czwickl @ASL-rmarshall :
The above reference is the template. See the description here: https://www.ema.europa.eu/en/human-regulatory-overview/research-development/paediatric-medicines-research-development/paediatric-investigation-plans
"A paediatric investigation plan (PIP) is a development plan aimed at ensuring that the necessary data are obtained through studies in children, to support the authorisation of a medicine for children. "
It clearly refers to studie**s** which means it is the same kind of level as a clinical development plan.

**Comment by BSnoeijerCD** [comment]
@daveih @ASL-rmarshall @czwickl @EMuhlbradt 
Based on our discussions yesterday, there are two different ways to include this plan ID concept. 

Option 1 is to reuse the identifier class. But as it is more general now that would imply you need to rename it from StudyIdentifier to Identifier:
![Identifiers_option1](https://github.com/user-attachments/assets/a82d8827-a59c-438b-99a6-c672c92a8c92)

Option 2 is to create a new class for reference Identifiers (naming tbd). Then we can leave the studyIdentifier class as is:
![Identifiers_option2](https://github.com/user-attachments/assets/f60117e8-5e27-46ed-801e-a427673744ee)

Although it changes the naming, I believe option 1 is more elegant and and efficient at the end. This makes it also possible to add more identifiers types anywhere in the model if needed.

**Comment by ASL-rmarshall** [comment]
@BSnoeijerCD I think we'd need scope for reference identifiers too (e.g., to specify EMA for a PIP).  I guess there might be an intermediate option 1.5 where:
-  `StudyIdentifier` and `ReferenceIdentifier` are both types of `Identifier`
- Only `ReferenceIdentifier` needs a `type`
- The `studyIdentifier` attribute would probably need a more generic name (e.g., `identifierText` as shown, but not highlighted)

![other identifiers](https://github.com/user-attachments/assets/e0a07aa5-41f4-417f-b46a-f32238a57e62)

**Comment by BSnoeijerCD** [comment]
@ASL-rmarshall @daveih 
I agree that identifierScope relationship should also be added for the referenceIdentifier.
The inheritance might make sense indeed and aligns with other inheritance parts of the model. That also makes the change smaller for users as the StudyIdentifier class name will not change. However the identifier attribute name and identifierScope reference name still needs to be changed (removing redundancy as we did in other parts of the model). 
Adding 'Text' to the identifier name could be confusing as that might imply you want a description instead of a number/letter combination. Though I can see where you are coming from as it now is identical to the class name.

**Comment by BSnoeijerCD** [comment]
@EMuhlbradt @czwickl @dih-cdisc @ASL-rmarshall : 
As discussed during the scrum today: please see below the updated UML for identifiers:<img width="1168" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM1ljQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--52071cd77ee3e2b9326ac5a9739c1ceaf4e0591d/IdentifiersAnnot.png" alt="IdentifiersAnnot.png" />

**Comment by EMuhlbradt** [comment]
> @EMuhlbradt @czwickl @dih-cdisc @ASL-rmarshall : As discussed during the scrum today: please see below the updated UML for identifiers: <img alt="IdentifiersAnnot.png" width="1168" src="https://camo.githubusercontent.com/382188d37fc738db046d7536d1eddf50659eceaa5650c4596a8cc70a4bc60d02/68747470733a2f2f6170692e7a656e6875622e636f6d2f617474616368656446696c65732f65794a66636d467062484d694f6e73696257567a6332466e5a534936496b4a42614842424d324e6a516d633950534973496d563463434936626e56736243776963485679496a6f69596d7876596c39705a434a3966513d3d2d2d666334383038363632633430396562653136653830346231356364396264386331653038383737322f4964656e74696669657273416e6e6f742e706e67">

@BSnoeijerCD I cannot open the png file for some reason. Can you send it to me in a different way?

**Comment by BSnoeijerCD** [comment]
@EMuhlbradt please find below again. It shows in zenhub but not in Github for some reason. I also send it by mail.
![IdentifiersAnnot](https://github.com/user-attachments/assets/98d67439-11f0-4485-bddb-e8504b54b8b9)

**Comment by BSnoeijerCD** [comment]
iscussed during scrum today: We will change attribute name to text instead of identfierText

**Comment by EMuhlbradt** [comment]
@BSnoeijerCD @dih-cdisc @ASL-rmarshall @czwickl please see the sprint 7 new and changes tabs with the proposed semantic changes to support this ticket.
[DDF.Terminology_Sprint.7.changes_2024-08-02.xlsx](https://github.com/user-attachments/files/16471642/DDF.Terminology_Sprint.7.changes_2024-08-02.xlsx)

**Comment by BSnoeijerCD** [comment]
@EMuhlbradt @czwickl : looks good. No comments. I aligned the UML regarding the text attribute and uploaded it to the github now.

**Comment by ASL-rmarshall** [comment]
@BSnoeijerCD @dih-cdisc @EMuhlbradt @czwickl : if we've renamed `identfierText` to `text`, is there a reason why we wouldn't rename:
- `Identifier.identifierScope` to `Identifier.scope` and, by inheritance, 
    - `StudyIdentifier.identifierScope` to `StudyIdentifier.scope` and
    - `ReferenceIdentifier.identifierScope` to `ReferenceIdentifier.scope`, and
- `Organization.organizationType` to `type`?

**Comment by dih-cdisc** [comment]
> @BSnoeijerCD @dih-cdisc @EMuhlbradt @czwickl : if we've renamed `identfierText` to `text`, is there a reason why we wouldn't rename:
> 
> * `Identifier.identifierScope` to `Identifier.scope` and, by inheritance,
>   
>   * `StudyIdentifier.identifierScope` to `StudyIdentifier.scope` and
>   * `ReferenceIdentifier.identifierScope` to `ReferenceIdentifier.scope`, and
> * `Organization.organizationType` to `type`?

I would not complain :)

**Comment by BSnoeijerCD** [comment]
> > @BSnoeijerCD @dih-cdisc @EMuhlbradt @czwickl : if we've renamed `identfierText` to `text`, is there a reason why we wouldn't rename:
> > 
> > * `Identifier.identifierScope` to `Identifier.scope` and, by inheritance,
> >   
> >   * `StudyIdentifier.identifierScope` to `StudyIdentifier.scope` and
> >   * `ReferenceIdentifier.identifierScope` to `ReferenceIdentifier.scope`, and
> > * `Organization.organizationType` to `type`?
> 
> I would not complain :)

There is no reason not to can change identifierScope to scope. This is in alignment with the rest of the changes we did before.

However, for organizationType we deliberately added 'organization' back to the attribute name in phase 3 because this class is also inherited by researchOrganization and then the meaning of type was changing when organization was not added. Therefore, when adding that part to the model we decided to add organization again to type to be absolutely clear and we could have a type of 'Research Organization'. Back then we also discussed to review this part of the model in phase 4 to make it more applicable to all organizations that might be included in a study design / protocol. See #293 We can then reconsider this part.
